packages/core/styles/components/_badge.scss

```
@@ -5,7 +5,7 @@ span[role='status'][data-intent] {
   font-size: var(--ix-size-sm);
   line-height: var(--ix-line-height);
 
-  padding: 0 var(--ix-space-md);
+  padding: 5px var(--ix-space-lg);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)